browser.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+module.exports = {
+	info: '\u{2139}\u{fe0f}',
+	success: '\u{2714}\u{fe0f}',
```

Comment:
Would be better to use `✅`

And use the actual characters instead of the unicode literals.